src/System.Runtime.Serialization.Xml/ref/project.json

```diff
@@ -3,7 +3,7 @@
     "System.Runtime": "4.0.0",
     "System.Xml.ReaderWriter": "4.0.0",
     "System.IO": "4.0.0",
-    "System.Runtime.Serialization.Primitives": "4.0.10",
+    "System.Runtime.Serialization.Primitives": "4.1.0-beta-23423",
```

Comment:
Do we want to reference this specific version of the package? Should this be "4.1.0-beta-*"?